proto/viam/app/build/v1/build.proto

```diff
@@ -23,3 +27,51 @@ message StartBuildResponse {
   // github's ID for the build
   optional string github_id = 1;
 }
+
+message GetLogsRequest {
```

Comment:
out of scope for this PR, but do we want to support paging / tailing / following behavior?

Reply:
As long as app doesn't send a end of stream, the client shouldn't close the connection

Reply:
(there might be a timeout I don't know about but this should work)